stacklet/client/sinistral/output.py

```diff
@@ -61,9 +103,19 @@ def on_execution_ended(self):
 
         sinistral = sinistral_client()
         scans_client = sinistral.client("scans")
+        ci_info = self.get_ci_info()
+
+        # We don't need to record CI info for local runs.
+        if ci_info["service"] == "local":
+            ci_info = None
+
```

Comment:
the `get_ci_info()` call could be made in the else clause, we don't need to try to fetch them running locally